examples/canvas/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+//! [Web-sys docs](https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.CanvasRenderingContext2d.html)
+//! [Web-sys example](https://rustwasm.github.io/wasm-bindgen/examples/2d-canvas.html)
+//! [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawWindow)
+
+#[macro_use]
+extern crate seed;
+use seed::prelude::*;
+
+// Model
+
+struct Model {}
+
+// Update
+
+#[derive(Clone)]
+enum Msg {}
+
+fn update(_msg: Msg, _model: &mut Model, _: &mut impl Orders<Msg>) {}
+
+// View
+
+fn view(_model: &Model) -> impl View<Msg> {
+    vec![
+        h1!["Example canvas"],
+        canvas![
+            attrs![
+                At::Id => "canvas",
```

Comment:
I think we should replace "canvas" here in `view` and in function `draw` with `const CANVAS_ID..` to teach users the best practices.